app/assets/javascripts/components/InvitationForm.coffee

```diff
@@ -40,21 +40,30 @@ invitationForm = React.createClass
         merge respondentEmail,
               className: if @shouldShowErrorFor 'respondentEmail' then 'invalid' else ''
               type: 'text'
-              placeholder: 'e-mail respondent'
+              required: 'required'
+      label
+        alt: "e-mail respondent"
+        placeholder: "e-mail respondent"
```

Comment:
Lijkt me hier wat netter om een React-component `Label` voor te maken zodat je alt en placeholder niet hoeft te dupliceren. Zeker omdat we deze stijl later misschien ook op andere formulieren gaan toepassen wel handig.